src/the_program_newloop.cpp

```diff
@@ -471,7 +472,13 @@ int c_the_program_newloop::main_execution() {
 	this->programtask_load_my_keys();
 	this->use_options_peerref();
 
-	c_linux_tuntap_obj tuntap;
+#ifdef __linux__
```

Comment:
please create platform.hpp that is included in all files and defines/undefines

ANTINET_windows
ANTINET_linux
ANTINET_macosx

also UNICODE / _UNICODE and other things that must be same for entire project